### DISCUSSION TITLE
Image optimization for Nextjs SSG markdown

### DISCUSSION THREAD

**Discussion by MejanH** [vercel/next.js#18574] [Help]
I know with Mdx is possible. But Mdx also created with Remark or Rehype plugins. I can use HTML tags in Markdown with Rehype. But how can I use the new NextJS Image Optimization?

**Comment by tseidler** [comment]
I read in the RFC thread that next/image + SSG was a non-goal. I take it this means the next/image optimization cannot be used on static sites?

(on my site it tries to access `_next/image/?url=.........`, but that doesn't exist ;-)).

Are there plans to support this, or is this simply not possible at this time?
